spec/factories/intakes.rb

```diff
@@ -298,6 +298,13 @@
     email_address { Faker::Internet.email }
   end
 
+  trait :with_address do
```

Comment:
@alaroia-cfa @caitbot You don't have to do the address setting during testing anymore.